packages/server-wallet/jest/chain-setup.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-process-env */
+import {GanacheServer} from '@statechannels/devtools';
+import {utils} from 'ethers';
+
+import config from '../src/config';
+import {deploy} from '../deployment/deploy';
+
+export default async function setup(): Promise<void> {
+  const account = {privateKey: config.serverPrivateKey, amount: utils.parseEther('100').toString()};
+  const ganacheServer = new GanacheServer(8545, 1337, [account]);
+  await ganacheServer.ready();
+
+  const deployedArtifacts = await deploy();
+
+  // TODO: is this the best way to add these addresses?
+  process.env.ETH_ASSET_HOLDER_ADDRESS = deployedArtifacts.ethAssetHolder;
+  process.env.ERC20_ADDRESS = deployedArtifacts.token;
+  process.env.ERC20_ASSET_HOLDER_ADDRESS = deployedArtifacts.tokenAssetHolder;
```

Comment:
You could name the artifacts in all-caps in the `deploy.ts` file, then:

`process.env = {...process.env, ...deployedArtifacts};`